crates/pixi_default_sys/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+authors.workspace = true
+description = "This crate defines the default system requirements"
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+name = "pixi_default_sys"
```

Comment:
Maybe the _sys suffix might instill the wrong idea since thats often used for system installed packages.

Reply:
`pixi_default_versions`?

Reply:
Yeah sounds good!

Reply:
Ok done